src/main/resources/ValidationMessages.properties

```diff
@@ -16,4 +16,5 @@ consultationWithStakeholdersSelection.selectionNotMade=Tell us if there has been
 directorsRemuneration.directorsRemuneration.missing=Enter the directors'' remuneration information
 directorsRemunerationSelection.selectionNotMade=Tell us if directors'' remuneration was received
 transferOfAssetsSelection.selectionNotMade=Tell us if there there any transfer of assets other than for full consideration
-transferOfAssets.transferOfAssets.missing=Enter details of any transfer of assets that have not reached market value
\ No newline at end of file
+transferOfAssets.transferOfAssets.missing=Enter details of any transfer of assets that have not reached market value
+payment.makePaymentChoice=Please choose whether to make a payment or not
```

Comment:
You'll need to run the wording choice past content designers. Typically, these statements start 'Tell us...'